src/core/ui.js

```diff
@@ -114,6 +114,58 @@ function createWarnButton(butName, arr, title) {
     ["label", `Document ${title.toLowerCase()}`],
   ]);
   ariaDecorate(button, ariaMap);
+
+
+  var numSteps = 20.0;
+  var boxElement
+  var prevRatio = 0.0;
+  var increasingColor = "rgba(255, 255, 255)";
+  var decreasingColor = "rgba(255, 255, 255, ratio)";
+
+  button.addEventListener("load", function (event) {
+    boxElement = document.querySelector("#respec-pill");
+
+    createObserver();
+  }, false);
+
+  function createObserver() {
+    var observer;
+
+    var options = {
+      root: null,
+      rootMargin: "0px",
+      threshold: buildThresholdList()
+    };
+
+    observer = new IntersectionObserver(handleIntersect, options);
+    observer.observe(boxElement);
+  }
+
+  function buildThresholdList() {
+    var thresholds = [];
+    var numSteps = 20;
+
+    for (var i = 1.0; i <= numSteps; i++) {
+      var ratio = i / numSteps;
+      thresholds.push(ratio);
+    }
+
+    thresholds.push(0);
+    return thresholds;
+  }
+
+  function handleIntersect(entries, observer) {
+    entries.forEach(function (entry) {
+      if (entry.intersectionRatio > prevRatio) {
+        entry.target.style.backgroundColor = increasingColor.replace("ratio", entry.intersectionRatio);
```

Comment:
`increasingColor` doesn't have `ratio` as a substring to replace. Maybe left there.